corehq/apps/analytics/tasks.py

```diff
@@ -212,7 +214,10 @@ def track_app_from_template_on_hubspot(webuser, cookies, meta):
 
 @task(queue="background_queue", acks_late=True, ignore_result=True)
 def track_clicked_deploy_on_hubspot(webuser, cookies, meta):
-    _send_form_to_hubspot(HUBSPOT_CLICKED_DEPLOY_FORM_ID, webuser, cookies, meta)
+    ab = {
+        'a_b_variable_deploy': 'A' if deterministic_random(webuser.username + 'a_b_variable_deploy') > 0.5 else 'B',
```

Comment:
Im hoping to pull all of these out of the code, in an overhaul of the way we set up AB tests, soon, but happy to pull this one out in the meantime if you prefer.